### DISCUSSION TITLE
automation question and squeezebox

### DISCUSSION THREAD

**Discussion by userMak** [music-assistant/support#1248] [Q&A] [answered]
Hi all,
I am on the beta version, and I would like to make an automation when I am talking on my phone to mute music assistant or even better to lower the volume. I am using squeezelite and I don't seem to have the option. Is there something I can do?

![image](https://github.com/music-assistant/hass-music-assistant/assets/55662933/1fad6a79-6b1f-4ec4-9f9c-23b07610f9cd)

**Comment by gieljnssns** [comment]
https://github.com/orgs/music-assistant/discussions/1193

The integration with Home Assistant isn't finished yet.

**Reply by userMak** [reply]
Yes, I know that. Do you know it will be available in the future then? For the time being I don't have a media player to handle..

**Reply by OzGav** [reply]
Yes it will be available in the future

**Reply by OzGav** [reply]
@userMak The integration which will allow automations is now available.

**Reply by userMak** [reply]
I updated MA but I can not a find the entity.. Where should I look? or what I should do?
if possible give me an example please.

**Reply by OzGav** [reply, marked as answer]
Have you added the new MA integration from HACS? See the instructions.

**Reply by userMak** [reply]
Ok I just added it from HACS and it seems good! I will test it later when I return home
Thanks!!

**Reply by OzGav** [reply]
Great. Please mark this an answered once you are happy

**Reply by userMak** [reply]
it is working very well!
thanks
